meinberlin/react/kiezradar/KiezradarMap.jsx

```diff
@@ -1,5 +1,145 @@
-import React from 'react'
+import L from 'leaflet'
+import React, { useState } from 'react'
+import django from 'django'
+import { Circle, ZoomControl } from 'react-leaflet'
+import { SearchAndShowAddress } from 'adhocracy4'
+import GeoJsonMarker from 'adhocracy4/adhocracy4/maps_react/static/a4maps_react/GeoJsonMarker'
+import ControlWrapper from 'adhocracy4/adhocracy4/maps_react/static/a4maps_react/ControlWrapper'
+import { Map } from '../contrib/map/Map'
+import { createPortal } from 'react-dom'
 
-export default function KiezradarMap () {
-  return <div className="kiezradar__map">Placeholder map</div>
+const chooseYourRadiusText = django.gettext('Choose your radius')
+
+export default function KiezradarMap ({
+  attribution,
+  bounds,
+  baseUrl,
+  mapboxToken,
+  omtToken,
+  center,
+  point,
+  radius,
+  minRadius,
+  maxRadius,
+  onChange
+}) {
+  return (
+    <div className="kiezradar__map">
+      <div id="mobile-controls" />
+      <Map
+        id="project-map"
+        zoomControl={false}
+        maxZoom={18}
+        attribution={attribution}
+        center={center}
+        bounds={bounds}
+        baseUrl={baseUrl}
+        mapboxToken={mapboxToken}
+        omtToken={omtToken}
+        style={{ minHeight: '100%', height: '100%' }}
+      >
+        <Radius
+          point={point}
+          radius={radius}
+          minRadius={minRadius}
+          maxRadius={maxRadius}
+          onChange={onChange}
+        />
+        <SearchAndShowAddress apiUrl="https://bplan-prod.liqd.net/api/addresses/" />
+        <ZoomControl position="topleft" />
+      </Map>
+    </div>
+  )
+}
+
+function Radius ({
+  point: _point,
+  radius: _radius,
+  minRadius,
+  maxRadius,
+  onChange
+}) {
+  const [point, setPoint] = useState(_point)
+  const [radius, setRadius] = useState(_radius)
+
+  const radiusControls = (
+    <RadiusControls
+      radius={radius}
+      minRadius={minRadius}
+      maxRadius={maxRadius}
+      onChange={(radius) => {
+        setRadius(radius)
+        onChange({ point, radius })
+      }}
+    />
+  )
+
+  return (
+    <>
+      <ControlWrapper position="topright">
+        <div className="kiezradar__controls--tablet">{radiusControls}</div>
+        {createPortal(
+          <div className="kiezradar__controls--mobile">{radiusControls}</div>,
+          document.getElementById('mobile-controls')
+        )}
+      </ControlWrapper>
+      <GeoJsonMarker
```

Comment:
You can also use a normal marker, no need to generate the whole geojson structure then. https://react-leaflet.js.org/docs/example-draggable-marker/